### PR TITLE
[TC-372] get and display different experiences

### PR DESCRIPTION
**Pull request by fw-noel**
[TC-372](https://bcdevex.atlassian.net/browse/TC-372)

Objective:
Display functions / experience levels in profile page
Designs from [Figma Prototype](https://www.figma.com/proto/EsuYOEyGPvIVDEFGnYCJ2f/CoCo-EMCR%3A-TEAMS-Redesign?page-id=2349%3A12933&type=design&node-id=2358-47218&viewport=-237%2C-1598%2C0.2&t=kxYM1ZJE08ZzOdWh-1&scaling=scale-down&starting-point-node-id=2358%3A47218)

Screenshot 2024-03-06 at 12 40 36 AM

[TC-372]: https://bcdevex.atlassian.net/browse/TC-372?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: backend/src/personnel/personnel.service.ts [hunk 1]
```diff
@@ -186,7 +186,8 @@ export class PersonnelService {
     role: Role,
     id: string,
   ): Promise<Record<string, PersonnelRO>> {
-    const person = await this.personnelRepository.findOneBy({ id: id });
+    const person = await this.personnelRepository.findOne({ where: { id }, relations: ['experiences', 'experiences.function'] });
+    console.log(person);
```
Comment (chelsea-EYDS) [inline backend/src/personnel/personnel.service.ts:190]: console.log

File: frontend/src/hooks/useFunctions.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import { AxiosPrivate } from '@/utils';
+import { useEffect, useState } from 'react';
+
+const useFunctions = () => {
+  const [functions, setFunctions] = useState<FunctionType[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const { data } = await AxiosPrivate.get('/function');
+      setFunctions(data);
```
Comment (chelsea-EYDS) [inline frontend/src/hooks/useFunctions.tsx:11]: Could we get this from the useGetFilters hook? (if prefer alone this is good too!)

File: frontend/src/pages/profile/ProfileFunctions.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+          />
+        );
+      case Experience.OUTSIDE_EXPERIENCED:
+        return (
+          <Chip
+            className="text-yellow bg-yellow border-darkYellow border-2 rounded-md font-bold capitalize py-0"
+            value="Outside Experience"
+          />
+        );
+      // f9f2f6 826521
```
Comment (chelsea-EYDS) [inline frontend/src/pages/profile/ProfileFunctions.tsx:45]: comment?

**Review by chelsea-EYDS — APPROVED** [review]

File: frontend/src/hooks/useFunctions.tsx (hunk 2 above)
Comment (fw-noel) [reply frontend/src/hooks/useFunctions.tsx:11]: No

**Review by chelsea-EYDS — APPROVED** [review]

**Pull request merged**